orttraining/orttraining/python/training/ortmodule/experimental/hierarchical_ortmodule/_hierarchical_ortmodule.py

```diff
@@ -1,14 +1,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-# debug_options.py
+# _hierarchical_ortmodule.py
 import tempfile
 import warnings
 
 import torch
 
 from .... import ortmodule
 from ... import ORTModule
-from ...debug_options import DebugOptions, LogLevel
+from ...options import DebugOptions, LogLevel
```

Comment:
Possible to use absolute import?

Reply:
The way the source code is laid out needs to match the import path. I think we can consider updating how the code is organized